### DISCUSSION TITLE
Fecha entrega TPI3

### DISCUSSION THREAD

**Discussion by agmblanco** [dds-jv/foro#434] [Consultas generales] [answered]
Buenas profes, cómo están? Tengo una duda sobre cuál sea la fecha correcta de entrega para el TPI3. De la planificación entiendo que se debe entregar el 10/07/2024. Sin embargo, en la [tarea de GitHub Classroom](https://classroom.github.com/assignment-invitations/cd2e4dcfa819f315b77d47b0bd2e6fec/status) se indica que la fecha de entrega es el 11/09/2024 por lo que no sé cuál debo considerar como válida.

Saludos cordiales, Agustín Blanco.

**Comment by flbulgarelli** [comment, marked as answer]
Hola! 

Se entrega el 10/7, como indica la planificación. Ya corregimos la tarea en Github, para que sea consistente con eso. 

Gracias por avisar

**Reply by agmblanco** [reply]
Perfecto, gracias!
